charts/kubernetes-external-secrets/templates/rbac.yaml

```diff
@@ -12,9 +15,11 @@ rules:
   - apiGroups: [""]
     resources: ["secrets"]
     verbs: ["create", "update"]
+{{- if not .Values.scopeNamespace.enabled }}
```

Comment:
I think we can omit this rule if namespaces are scoped, but I'm not sure on how the app works

Reply:
We probably still need to be able to check the namespace we are running in since the user can put annotations on the namespace to configure allowed iam roles or key naming patterns

Reply:
That's a good point - shall I leave it as it was, or limit it using `resourceNames`?
```  - apiGroups: [""]
    resources: ["namespaces"]
    resourceNames: [{{ .Release.Namespace | quote }}]
    verbs: ["get", "watch", "list"]
```